src/resources/css/docs.css

```diff
@@ -133,26 +133,24 @@ main nav li a {
 	color: #546c75;
 }
 
-
-
```

Comment:
These extra empty lines were kinda intentional to split apart the file in sections. We should probably add comments between these sections instead I figure.

Reply:
Thanks, I'm happy to add a note here, do you know what note I should add? It looks like more general selectors are above and more specific ones are below, but I'm guessing.